libnopegl/src/nopegl.h.in

```diff
@@ -720,6 +720,8 @@ struct ngl_config {
     const char *hud_export_filename; /* Path to the HUD export file (CSV). Disables display if enabled. */
 
     int hud_scale;           /* Scaling applied to the HUD, useful for high DPI displays */
+
+    int debug; /* Enable graphics context debugging */
```

Comment:
Suggestion for later: we already have a `hud` field, which we could change to `u32 debug` with a bunch of flags:

- `NGL_DEBUG_HUD`: enable HUD debugging
- `NGL_DEBUG_GRAPHICS`: enable graphics context debugging
- `NGL_DEBUG_GRAPHICS_ASSERT_ON_ERROR`: assert on graphics context error but does tolerate shader miscompilation
- `NGL_DEBUG_GRAPHICS_ASSERT_ON_ERROR_STRICT`: assert on any graphics context error

Reply:
Sounds reasonable.